lib/matplotlib/markers.py

```
@@ -405,88 +460,136 @@ def _get_hexagon2(self):
+        if fs == 'full':
+            self._path = polypath
+        else:
+            # TODO: Implement partially-filled octagons
+            self._path = polypath
+            
+    _line_marker_path = Path([[0.0, -1.0], [0.0, 1.0]])
+    def _set_vline(self):
+        self._transform = Affine2D().scale(0.5)
+        self._snap_threshold = 1.0
+        self._filled = False
+        self._path = _line_marker_path
```

Ruff (0.12.2)

Undefined name `_line_marker_path`

(F821)